src/github.js

```diff
@@ -89,7 +89,7 @@ async function createPrComment(repoToken, markupData, updateCommentIfOneExists)
     let existingCommentId = null;
     if (updateCommentIfOneExists) {
       core.info('Checking for existing comment on PR....');
-      existingCommentId = await lookForExistingComment(octokit);
+      existingCommentId = await lookForExistingComment(octokit, commentIdentifier);
     }
 
     let response;
```

Comment:
Below this section there are two places where markupPrefix is used.  Since the ```<!--```  and ```-->``` have been removed from the markupPrefix variable at the top of this file they will need to be included in the usages below (around lines 102 and 111.  

It might be better to declare the variable inside of this function instead of at the top of the file that way you have the comment portion, the prefix and the commentIdentifier all in one place instead of having to remember to format it that way in multiple places.

Reply:
I moved that as recommended and setup the markupPrefix once to be used and passed along.